config/role_map.yml

```diff
@@ -3,6 +3,7 @@ development:
     - aweadadmin
     - scossuadmin
     - citi
+    - laketest
```

Comment:
whoops, not sure if this can go in here. Sent email to @awead about changing role_map.yml[:development] or if this file should not go in version control, but rather be ~/.globallygitignored